planner/core/planbuilder.go

```diff
@@ -1605,6 +1605,10 @@ func (b *PlanBuilder) buildShow(ctx context.Context, show *ast.ShowStmt) (Plan,
 	case ast.ShowCreateView:
 		err := ErrSpecificAccessDenied.GenWithStackByArgs("SHOW VIEW")
 		b.visitInfo = appendVisitInfo(b.visitInfo, mysql.ShowViewPriv, show.Table.Schema.L, show.Table.Name.L, "", err)
+	case ast.ShowTableNextRowId:
+		p := &ShowNextRowID{TableName: show.Table}
```

Comment:
I think need to check privilege, just like `show create table`

```go
		b.visitInfo = appendVisitInfo(b.visitInfo, mysql.AllPrivMask, show.Table.Schema.L, show.Table.Name.L, "", err)
```

Reply:
> @zhang555 Thanks for your contribution, can you fill a parser PR too, and link with this PR.

parser have been merged . 

https://github.com/pingcap/parser/pull/738

Reply:
see https://github.com/pingcap/tidb/issues/14567 . 

`admin show $table_name next_row_id`  need super privilege. 

`show table $table_name next_row_id`  only use `SELECT` privilege will be ok .

Reply:
Ye, but still need privilege check for `SelectPriv` here, there is no check now.

Reply:
@crazycs520  done